fdbcli/GetAuditStatusCommand.actor.cpp

```diff
@@ -45,11 +45,18 @@ ACTOR Future<bool> getAuditStatusCommandActor(Database cx, std::vector<StringRef
 	}
 
 	if (tokencmp(tokens[2], "id")) {
+		if (tokens.size() != 4) {
+			printUsage(tokens[0]);
+			return false;
+		}
 		const UID id = UID::fromString(tokens[3].toString());
 		AuditStorageState res = wait(getAuditState(cx, type, id));
 		printf("Audit result is:\n%s", res.toString().c_str());
 	} else if (tokencmp(tokens[2], "recent")) {
-		const int count = std::stoi(tokens[3].toString());
+		int count = 5;
```

Comment:
What is this? 5 'rows'? We should add this to the help output, that 5 is the default 'Count' (I can do that over in #9266).

Reply:
I think default should be ROW_LIMIT_UNLIMITED -- I think that would make more sense to the user. What do you think ?

Reply:
That's a good point, I put `CLIENT_KNOBS->TOO_MANY` as the default.

Reply:
Is TOO_MANY too small? Its only 1M? Thanks.

Reply:
My current plan is to keep only the most recent `N` audit records per audit type, I feel `N` would be smaller than 1M, and a background process will be added to GC the earlier audit results.